testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/builder/AbstractBuilderTest.java

```diff
@@ -267,7 +280,7 @@ protected ResourceSet getResourceSet(IProject project) {
 	 */
 	protected void assertXtextIndexIsValid() {
 		// ensure no build is running while we examine the Xtext index
-		final ISchedulingRule rule = externalLibraryBuilderHelper.getRule();
+		final ISchedulingRule rule = ResourcesPlugin.getWorkspace().getRoot();
```

Comment:
why is it called "getRoot"?

Reply:
The workspace root is the logical parent of all projects in the workspace. I don't really know why they made a distinction between workspace and workspace root. Probably to decouple the workspace from the IResource concept.